src/EPiServer.SocialAlloy.Web/Social/Repositories/Ratings/PageRatingRepository.cs

```diff
@@ -76,24 +80,35 @@ public void AddRating(string user, string target, int value)
         /// the Social cloud services.</exception>
         public int? GetRating(PageRatingFilter filter)
         {
-            int? result = null;
+            var filters = new List<FilterExpression>();
+
+            if (!string.IsNullOrWhiteSpace(filter.Rater))
+            {
+                this.ratingFilters.Rater.EqualTo(Reference.Create(filter.Rater));
```

Comment:
The filters are now being conditionally created but they aren't being assigned to anything. Need a filters.Add(...) around the expressions being created here.